scripts/marketing_data_test.py

```diff
@@ -318,14 +337,18 @@ def marketing_dml_test(
 
         logger.info("#############################################################################")
 
-        # logger.info(f"Fitting the standard (confounded) model with params {y_params}")
-        # noncausal_est = XGBClassAUCScore(**y_params)
-        # noncausal_est.fit(pd.concat([X,T], axis=1), y)
-        # logger.info("Standard model fit done")
-        # standard_predictions = pd.DataFrame(noncausal_est.predict_proba(treat_est_combo)[:,1])
-        # standard_predictions = pd.concat([standard_predictions,treat_identifiers],axis=1)
-        # logger.info(f"Non-causal model fit is done")
-        # logger.info(f'{pd.DataFrame(standard_predictions).describe()}')
+        logger.info(f"Fitting the standard (confounded) model with params {y_params}")
+        noncausal_est = XGBClassAUCScore(**y_params)
+        # TODO: Confirm with Carl that redefining treat_est_combo is fine since we removed
+        # collinear features.
+        treat_est_combo = pd.concat([X,T], axis=1)
```

Comment:
Assign `pd.concat([X,T], axis=1)` as `treat_est_combo` since this is post collinearity removal. In your commented lines above, you have both pre and post removals.